deb/brave-keyring/DEBIAN/control

```diff
@@ -2,6 +2,5 @@ Package: brave-keyring
 Architecture: all
 Maintainer: Ben Kero <bkero@brave.com>
 Priority: optional
-Version: 1.10
+Version: 1.11
 Description: Brave Browser keyring and repository files
-Depends: gnupg, procps
```

Comment:
@bkero / @fmarier seems we don't need gnupg anymore. Not sure what procps was needed for, but it also seems unnecessary. Unless I'm missing something.

Reply:
https://github.com/brave/brave-release/commit/adc029f90bdc54e726c9b1d0f3ee585f8460aaef#diff-ebb66baa63687766d559524c51821e3ed81ac16e72a8cd580c1b4c2c1f4f9922

Reply:
tl;dr we use this for ensuring sysctl is installed for some sysctls we are setting. @fmarier still recommended?

Reply:
Yes, it looks like it's no longer necessary since we're not setting the `sysctl` settings anymore.

Reply:
Sounds good, this can be removed then